documentation/source/overview/studies.rst

```diff
@@ -1,120 +1,8 @@
-   * - ``sct_register_multimodal --param algo=slicereg``
-     - `Cohen-Adad et al. Slice-by-slice regularized registration for spinal cord MRI: SliceReg. Proc ISMRM 2015 <https://www.dropbox.com/s/v3bb3etbq4gb1l1/cohenadad_ismrm15_slicereg.pdf?dl=0>`__
-   * - ``sct_register_to_template``
-     - `De Leener B, Fonov VS, Louis Collins D, Callot V, Stikov N, Cohen-Adad J. PAM50: Unbiased multimodal template of the brainstem and spinal cord aligned with the ICBM152 space. Neuroimage 2017. <http://www.sciencedirect.com/science/article/pii/S1053811917308686>`__
-   * - ``sct_register_to_template --param algo=slicereg``
-     - `Cohen-Adad et al. Slice-by-slice regularized registration for spinal cord MRI: SliceReg. Proc ISMRM 2015 <https://www.dropbox.com/s/v3bb3etbq4gb1l1/cohenadad_ismrm15_slicereg.pdf?dl=0>`__
-   * - ``sct_straighten_spinalcord``
-     - `De Leener B et al. Topologically-preserving straightening of spinal cord MRI. J Magn Reson Imaging 2017 <https://www.ncbi.nlm.nih.gov/pubmed/28130805>`__
-
-Template and Atlas
-^^^^^^^^^^^^^^^^^^
-
-The table below provides references relevant to the :ref:`pam50` used by SCT, including a reference for the template itself, as well as earlier works that the template builds on.
-
-.. list-table::
-   :widths: 20 80
-   :header-rows: 1
-
-   * - Template/atlas
-     - References
-   * - PAM50 template
-     - `De Leener B, Fonov VS, Louis Collins D, Callot V, Stikov N, Cohen-Adad J. PAM50: Unbiased multimodal template of the brainstem and spinal cord aligned with the ICBM152 space. Neuroimage 2017. <http://www.sciencedirect.com/science/article/pii/S1053811917308686>`__
-   * - MNI-Poly-AMU template
-     - `Fonov et al. Framework for integrated MRI average of the spinal cord white and gray matter: The MNI-Poly-AMU template. Neuroimage 2014. <https://www.ncbi.nlm.nih.gov/pubmed/25204864>`__
-   * - White matter atlas
-     - `Lévy et al. White matter atlas of the human spinal cord with estimation of partial volume effect. Neuroimage 2015 <https://www.ncbi.nlm.nih.gov/pubmed/26099457>`__
-   * - Probabilistic atlas (AMU40)
-     - `Taso et al. A reliable spatially normalized template of the human spinal cord–Applications to automated white matter/gray matter segmentation and tensor-based morphometry (TBM) mapping of gray matter alterations occurring with age. Neuroimage 2015 <https://www.ncbi.nlm.nih.gov/pubmed/26003856>`__
-   * - Spinal levels
-     - `Cadotte DW, Cadotte A, Cohen-Adad J, Fleet D, Livne M, Wilson JR, Mikulis D, Nugaeva N, Fehlings MG. Characterizing the location of spinal and vertebral levels in the human cervical spinal cord. AJNR Am J Neuroradiol, 2015, 36(4):803-810. <https://paperpile.com/app/p/5b580317-6921-06c8-a2ee-685d4dbaa44c>`_
-
-
-.. _references-applications:
-
-Applications
-------------
 The following studies (in chronological order) have used SCT:
```

Comment:
Is this list manually generated, or by a script? I find it to not be the most easily readable, perhaps we could put it in a table or something?

Reply:
> Is this list manually generated, or by a script?

It's currently manually generated, and each time a new reference is added by hand. (See [this search](https://github.com/neuropoly/spinalcordtoolbox/pulls?q=is%3Apr+%22add+reference%22) for examples)

(Also, just for context, there's currently an open PR for automating the references: #3170)

> I find it to not be the most easily readable, perhaps we could put it in a table or something?

Hmm... yeah, it is a bit like a "wall of text" right now. Using a table would certainly make each field (date, journal, author, title) much easier to read.

@jcohenadad Are there any reasons we would need to adhere to a specific reference format?

Reply:
a table with columns for authors, date, etc. is awkward (not common practice in academia)-- but i definitely agree we could pimp this up --> #3357